soc/arm/nuvoton_npcx/common/soc_power.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_NPCX_SOC_POWER_H_
+#define _NUVOTON_NPCX_SOC_POWER_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Receive whether the module for the console is in use.
+ *
+ * @return 1 if console is in use. Otherwise
+ * @return
+ *  - True if the console is in use.
+ *  - False otherwise,  the module for console is reday to enter low power mode.
+ */
+bool npcx_power_console_is_in_use(void);
+
+/**
+ * @brief Notify the power module that the module for the console is in use.
+ *
+ * Notify the power module that the module for the console is in use. It also
+ * extends expired time by CONFIG_SOC_POWER_CONSOLE_EXPIRED_TIMEOUT.
+ */
+void npcx_power_console_is_in_use_refresh(void);
+
```

Comment:
Can you also add an API for changing the console activity timeout?   See https://chromium.googlesource.com/chromiumos/platform/ec/+/cfbbbb93/chip/npcx/clock.c#474 for the expected usage.

Reply:
Done.